roles/pulp-devel/tasks/add_fixtures.yaml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Enable unprivileged_userns_clone
+  sysctl:
+    name: kernel.unprivileged_userns_clone
+    value: "1"
+  become: true
+
+- name: Pull the latest pulp-fixtures image
+  podman_image:
+    name: quay.io/pulp/pulp-fixtures
+    tag: latest
+  become: true
+  become_user: '{{ pulp_user }}'
```

Comment:
On a second thought, maybe this (convenience-)task should not be here / should be skippable.
At least in the ci, i think downloading that huge container is a pain.

Reply:
I am in increasing agreement. This is a lot more trouble than it is worth, and the `pfixtures` command should pull the image if it is not there already.

Reply:
I dropped the task.